e2e/cypress/support/ui_commands.js

```diff
@@ -432,3 +446,54 @@ Cypress.Commands.add('checkRunLDAPSync', () => {
         }
     });
 });
+
+Cypress.Commands.add('createArchivedChannel', (options, messages, memberIds) => {
```

Comment:
Maybe this could be split into two commands for `uiCreateChannel` and `uiArchiveChannel`? I'm surprised we don't actually have a `uiCreateChannel` already

Reply:
Ah, yeah I think it makes sense to add those two commands. Also, I'd suggest to make it specific with the UI flow so it can be used in other places. If it's a combination of everything - UI flow and several API calls - and to be used by particular spec file then I'd suggest not to make `createArchivedChannel` as command but a helper function in the spec file itself.

Reply:
most likely since there is an apiCreate, most people just go with that. But sure, I'll recreate this as a helper function in the spec and have uiCreateChannel and uiArchiveChannel as commands.